### DISCUSSION TITLE
About button binding

### DISCUSSION THREAD

**Discussion by mimimiaomiao1** [canton7/Stylet#300] [Q&A]
View.ActionTarget not set on control System.Windows.Controls.Button:  (method Login). This probably means the control hasn't inherited it from a parent, e.g. because a ContextMenu or Popup sits in the visual tree. You will need so set 's:View.ActionTarget' explicitly. See the wiki section "Actions" for more details.”

<Button Command="{s:Action Login}" />
The main page can be used. Sub pages cannot be used...

**Comment by mimimiaomiao1** [comment]
`<Button Content="{Binding SwitchLoginPageContent,Mode=TwoWay}" Command="{s:Action Login}"
The main page can be used. Sub pages cannot be used

 `

**Comment by canton7** [comment]
Please post a full sample which reproduces your issue. I don't know what "sub psages cannot be used" means.
